napari/_vispy/experimental/texture_atlas.py

```diff
@@ -2,20 +2,94 @@
 
 A texture atlas is a large texture that stores many smaller texture tiles.
 """
-from collections import namedtuple
-from typing import Optional, Tuple
+from typing import NamedTuple, Optional, Tuple
 
 import numpy as np
 from vispy.gloo import Texture2D
 
+from ...layers.image.experimental.octree_util import ChunkData
```

Comment:
As noted in your comment at the top of the PR, in the long run I think we won't want `ChunkData` here, but I'm fine to postpone until we have a complete solution working